app/utils/loadForm.js

```diff
@@ -0,0 +1,12 @@
+const loadForm = async (
```

Comment:
now that this isn't local to anything i think it could do with a rename, maybe loadFormAsStore?